### ISSUE TITLE
Shairport-sync failed to start 

### ISSUE THREAD

**Issue by Soundwavetm2** [mikebrady/shairport-sync#829]
I have a new install of shairport-sync.  Currently when i run the service it is showing code=exited, status=1/Failure  states failed with result 'exit-code'

I am trying to find out why this is not starting properly

**Comment by Soundwavetm2** [comment]
here is the status of the service 

● shairport-sync.service - Shairport Sync - AirPlay Audio Receiver
   Loaded: loaded (/lib/systemd/system/shairport-sync.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Wed 2019-04-03 13:30:03 EDT; 9min ago
  Process: 1315 ExecStart=/usr/local/bin/shairport-sync (code=exited, status=1/FAILURE)
 Main PID: 1315 (code=exited, status=1/FAILURE)

Apr 03 13:30:03 raspberrypi systemd[1]: Started Shairport Sync - AirPlay Audio Receiver.
Apr 03 13:30:03 raspberrypi systemd[1]: shairport-sync.service: Main process exited, code=exite
Apr 03 13:30:03 raspberrypi systemd[1]: shairport-sync.service: Unit entered failed state.
Apr 03 13:30:03 raspberrypi systemd[1]: shairport-sync.service: Failed with result 'exit-code'.

**Comment by mikebrady** [comment]
Hmm, that's curious alright. Could you edit the configuration file at `/etc/shairport-sync.conf` and set verbosity to `1` in the diagnostic section at the end of the file (remember to uncomment it!) and reboot. Then let use see the status report. Hopefully there will be more detail in it. What kind of machine / Linux is it running on? Also, if you could post the response to:
```
$ shairport-sync -V
```
it would be very useful. There's more [here](https://github.com/mikebrady/shairport-sync/blob/master/CONTRIBUTING.md).

**Comment by mikebrady** [comment]
Closing this. Please feel free to open another issue if necessary.

**Comment by jvyden420** [comment]
@mikebrady Mind reopening this? I think I've got a similar situation here. I'm using a rpi aswell.
```pi@jvy-headphones:~ $ sudo service shairport-sync status
● shairport-sync.service - ShairportSync AirTunes receiver
   Loaded: loaded (/lib/systemd/system/shairport-sync.service; enabled; vendor preset: enabled)
   Active: failed (Result: signal) since Sun 2019-09-08 03:30:11 BST; 2min 26s ago
     Docs: man:shairport-sync(7)
           file:///usr/share/doc/shairport-sync/README.md.gz
           https://github.com/mikebrady/shairport-sync
  Process: 480 ExecStart=/usr/bin/shairport-sync --daemon $DAEMON_ARGS (code=killed, signal=ABRT)

Sep 08 03:30:11 jvy-headphones systemd[1]: shairport-sync.service: Service RestartSec=100ms expired, scheduling restart.
Sep 08 03:30:11 jvy-headphones systemd[1]: shairport-sync.service: Scheduled restart job, restart counter is at 5.
Sep 08 03:30:11 jvy-headphones systemd[1]: Stopped ShairportSync AirTunes receiver.
Sep 08 03:30:11 jvy-headphones systemd[1]: shairport-sync.service: Start request repeated too quickly.
Sep 08 03:30:11 jvy-headphones systemd[1]: shairport-sync.service: Failed with result 'signal'.
Sep 08 03:30:11 jvy-headphones systemd[1]: Failed to start ShairportSync AirTunes receiver.
```
Strangely enough, running it directly in the terminal without systemd seems to work just fine.
Here's how it looks with verbosity.
```pi@jvy-headphones:~ $ sudo service shairport-sync status
● shairport-sync.service - ShairportSync AirTunes receiver
   Loaded: loaded (/lib/systemd/system/shairport-sync.service; enabled; vendor preset: enabled)
   Active: deactivating (stop-sigterm) (Result: signal)
     Docs: man:shairport-sync(7)
           file:///usr/share/doc/shairport-sync/README.md.gz
           https://github.com/mikebrady/shairport-sync
  Process: 1289 ExecStart=/usr/bin/shairport-sync --daemon $DAEMON_ARGS (code=killed, signal=ABRT)
    Tasks: 1 (limit: 2200)
   Memory: 1.0M
   CGroup: /system.slice/shairport-sync.service
           └─1291 /usr/bin/shairport-sync --daemon

Sep 08 03:34:05 jvy-headphones shairport-sync[1291]: metadata socket packet size is "500".
Sep 08 03:34:05 jvy-headphones shairport-sync[1291]: get-coverart is 0.
Sep 08 03:34:05 jvy-headphones shairport-sync[1291]: convolution is 0.
Sep 08 03:34:05 jvy-headphones shairport-sync[1291]: convolution IR file is "(null)"
Sep 08 03:34:05 jvy-headphones shairport-sync[1291]: convolution max length 8192
Sep 08 03:34:05 jvy-headphones shairport-sync[1291]: convolution gain is 0.000000
Sep 08 03:34:05 jvy-headphones shairport-sync[1291]: loudness is 0.
Sep 08 03:34:05 jvy-headphones shairport-sync[1291]: loudness reference level is -20.000000
Sep 08 03:34:05 jvy-headphones shairport-sync[1291]: disable resend requests is off.
Sep 08 03:34:05 jvy-headphones shairport-sync[1291]: diagnostic_drop_packet_fraction is 0.000000. A value of 0.0 means no packets will be dropped deliberately.
```

**Comment by mikebrady** [comment]
Thanks for the post. If you set the verbosity to 1, restart (`$ sudo systemctl restart shairport-sync`) and post the log here, that would be very useful. Also, if you could post the type of Raspberry Pi (from `$ cat /proc/device-tree/model`) and the version of the Operating System (from `$cat /etc/os-release`) they would be useful pieces of information.

**Comment by jvyden420** [comment]
Absolutely.
/proc/device-tree/model: 
`Raspberry Pi 3 Model B Rev 1.2`
/etc/os-release: 
```
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
```
journalctl -xe:
```Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: decoders_supported field is 1.
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: use_apple_decoder is 0.
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: alsa_use_hardware_mute is 0.
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: no special mdns service interface was requested.
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: configuration file name "/etc/shairport-sync.conf" resolves to "/etc/shairport-sync.conf".
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: metadata enabled is 0.
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: metadata pipename is "(null)".
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: metadata socket address is "(null)" port 0.
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: metadata socket packet size is "500".
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: get-coverart is 0.
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: convolution is 0.
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: convolution IR file is "(null)"
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: convolution max length 8192
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: convolution gain is 0.000000
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: loudness is 0.
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: loudness reference level is -20.000000
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: disable resend requests is off.
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: diagnostic_drop_packet_fraction is 0.000000. A value of 0.0 means no packets will be dropped deliberately.
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: shutdown requested...
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: Request to shut down all rtsp conversation threads
Sep 08 19:50:23 jvy-headphones shairport-sync[1203]: asking playing threads to stop
Sep 08 19:50:24 jvy-headphones systemd[1]: shairport-sync.service: Service RestartSec=100ms expired, scheduling restart.
Sep 08 19:50:24 jvy-headphones systemd[1]: shairport-sync.service: Scheduled restart job, restart counter is at 5.
-- Subject: Automatic restarting of a unit has been scheduled
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- 
-- Automatic restarting of the unit shairport-sync.service has been scheduled, as the result for
-- the configured Restart= setting for the unit.
Sep 08 19:50:24 jvy-headphones systemd[1]: Stopped ShairportSync AirTunes receiver.
-- Subject: A stop job for unit shairport-sync.service has finished
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- 
-- A stop job for unit shairport-sync.service has finished.
-- 
-- The job identifier is 1465 and the job result is done.
Sep 08 19:50:24 jvy-headphones systemd[1]: shairport-sync.service: Start request repeated too quickly.
Sep 08 19:50:24 jvy-headphones systemd[1]: shairport-sync.service: Failed with result 'signal'.
-- Subject: Unit failed
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- 
-- The unit shairport-sync.service has entered the 'failed' state with result 'signal'.
Sep 08 19:50:24 jvy-headphones systemd[1]: Failed to start ShairportSync AirTunes receiver.
-- Subject: A start job for unit shairport-sync.service has failed
-- Defined-By: systemd
-- Support: https://www.debian.org/support
-- 
-- A start job for unit shairport-sync.service has finished with a failure.
-- 
-- The job identifier is 1465 and the job result is failed.
```

**Comment by mikebrady** [comment]
Thanks – that's all useful. It's really peculiar that Shairport Sync itself appears to be shutting itself down. Unfortunately, the early part of the log is missing, and some of it might be useful. I'd be looking for something like this:

```
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.001115149|Started!
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000307342|software version: "3.3.2d5-OpenSSL-Avahi-ALSA-soxr-metadata-dbus-sysconfdir:/etc"
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000032917|log verbosity is 1.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000052083|alsa: alsa_maximum_stall_time of 0.200000 sec.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000056874|alsa: disable_standby_mode is "never".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000018698|alsa: disable_standby_mode_silence_threshold is 0.040000 seconds.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000018854|alsa: disable_standby_mode_silence_scan_interval is 0.004000 seconds.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000026094|alsa: output device name is "hw:Device".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000231925|disable resend requests is off.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000030573|diagnostic_drop_packet_fraction is 0.000000. A value of 0.0 means no packets will be dropped deliberately.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000027969|statistics_requester status is 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017760|rtsp listening port is 5000.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016927|udp base port is 6001.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017187|udp port range is 10.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016667|player name is "RaspberryPi3BP".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016823|backend is "(null)".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017083|run_this_before_play_begins action is "(null)".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016927|run_this_after_play_ends action is "(null)".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000020677|wait-cmd status is 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017343|run_this_before_play_begins may return output is 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017084|run_this_if_an_unfixable_error_is_detected action is "(null)".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016979|run_this_before_entering_active_state action is  "(null)".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017500|run_this_after_exiting_active_state action is  "(null)".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000020416|active_state_timeout is  10.000000 seconds.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000021823|mdns backend "(null)".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017343|interpolation setting is "auto".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016927|interpolation soxr_delay_threshold is 30.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017084|resync time is 0.050000 seconds.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000018020|allow a session to be interrupted: 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016771|busy timeout time is 120.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017083|drift tolerance is 0.001995 seconds.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000018386|password is "(null)".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016770|ignore_volume_control is 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016875|volume_max_db is not set
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000019740|volume range in dB (zero means use the range specified by the mixer): 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000018125|volume_range_combined_hardware_priority (1 means hardware mixer attenuation is used first) is 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017291|playback_mode is 0 (0-stereo, 1-mono, 1-reverse_stereo, 2-both_left, 3-both_right).
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017448|disable_synchronization is 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017031|use_mmap_if_available is 1.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017135|output_format automatic selection is enabled.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016875|output_rate automatic selection is enabled.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016823|audio backend desired buffer length is 0.200000 seconds.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000018021|audio_backend_buffer_interpolation_threshold_in_seconds is 0.120000 seconds.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000018385|audio backend latency offset is 0.000000 seconds.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000021406|audio backend silence lead-in time is -1.000000 seconds. A value -1.0 means use the default.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000018958|zeroconf regtype is "_raop._tcp".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017396|decoders_supported field is 1.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017083|use_apple_decoder is 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016823|alsa_use_hardware_mute is 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017031|no special mdns service interface was requested.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000037083|configuration file name "/etc/shairport-sync.conf" resolves to "/etc/shairport-sync.conf".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000020000|metadata enabled is 1.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017813|metadata pipename is "/tmp/shairport-sync-metadata".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017083|metadata socket address is "(null)" port 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000017448|metadata socket packet size is "500".
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016718|get-coverart is 1.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016823|loudness is 0.
Sep 08 23:31:03 raspberryPi3BP shairport-sync[4823]:          0.000016302|loudness reference level is -20.000000
Sep 08 23:31:05 raspberryPi3BP shairport-sync[4823]:          1.477307647|"soxr" interpolation has been chosen.
Sep 08 23:31:07 raspberryPi3BP sudo[4837]:       pi : TTY=pts/0 ; PWD=/home/pi ; USER=root ; COMMAND=/bin/journalctl -f -n 100
```

Could you see if you could fish it out, please? You might have to search a bit further back through the logs.

**Comment by jvyden420** [comment]
I don't really know where to find that. Should i try running with --daemon?

**Comment by mikebrady** [comment]
No. If you use a command like this:
```
$ sudo journalctl -n 300 -f
```
... it will log the last 300 lines of log entries (`-n 300`)  and will follow (`-f`) i.e. print new log entries as they occur. You can control-c out of it, or simply omit the `-f`.

**Comment by purrloftruth** [comment]
For me, running shairport-sync installed through apt on a Raspberry Pi 4 with a fresh installation of Raspbian Lite (fully updated), I get this issue when configuring the `name` value in the config. When I comment it out it's able to start fine.

I haven't tested to see if any other config variables give it trouble. (Log verbosity seems to work...)
```
Sep 09 17:59:50 cherry systemd[1]: Starting ShairportSync AirTunes receiver...
Sep 09 17:59:50 cherry shairport-sync[1418]: free(): double free detected in tcache 2
Sep 09 17:59:50 cherry shairport-sync[1420]: alsa output device name is "default".
Sep 09 17:59:50 cherry shairport-sync[1420]: Version: "3.2.2-OpenSSL-Avahi-ALSA-pa-dummy-stdout-pipe-soxr-convolution-metadata-sysconfdir:/etc"
Sep 09 17:59:50 cherry systemd[1]: shairport-sync.service: Control process exited, code=killed, status=6/ABRT
Sep 09 17:59:50 cherry shairport-sync[1420]: statistics_requester status is 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: daemon status is 1.
Sep 09 17:59:50 cherry shairport-sync[1420]: daemon pid file path is "/run/shairport-sync/shairport-sync.pid".
Sep 09 17:59:50 cherry shairport-sync[1420]: rtsp listening port is 5000.
Sep 09 17:59:50 cherry shairport-sync[1420]: udp base port is 6001.
Sep 09 17:59:50 cherry shairport-sync[1420]: udp port range is 100.
Sep 09 17:59:50 cherry shairport-sync[1420]: player name is "Stereo".
Sep 09 17:59:50 cherry shairport-sync[1420]: backend is "(null)".
Sep 09 17:59:50 cherry shairport-sync[1420]: on-start action is "(null)".
Sep 09 17:59:50 cherry shairport-sync[1420]: on-stop action is "(null)".
Sep 09 17:59:50 cherry systemd[1]: shairport-sync.service: Failed with result 'signal'.
Sep 09 17:59:50 cherry shairport-sync[1420]: wait-cmd status is 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: on-start returns output is 0.
Sep 09 17:59:50 cherry systemd[1]: Failed to start ShairportSync AirTunes receiver.
Sep 09 17:59:50 cherry shairport-sync[1420]: mdns backend "(null)".
Sep 09 17:59:50 cherry shairport-sync[1420]: stuffing option is "0" (0-basic, 1-soxr).
Sep 09 17:59:50 cherry shairport-sync[1420]: resync time is 0.050000 seconds.
Sep 09 17:59:50 cherry shairport-sync[1420]: allow a session to be interrupted: 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: busy timeout time is 120.
Sep 09 17:59:50 cherry shairport-sync[1420]: drift tolerance is 0.001995 seconds.
Sep 09 17:59:50 cherry shairport-sync[1420]: password is "(null)".
Sep 09 17:59:50 cherry shairport-sync[1420]: ignore_volume_control is 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: volume_max_db is not set
Sep 09 17:59:50 cherry shairport-sync[1420]: playback_mode is 0 (0-stereo, 1-mono, 1-reverse_stereo, 2-both_left, 3-both_right).
Sep 09 17:59:50 cherry shairport-sync[1420]: disable_synchronization is 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: use_mmap_if_available is 1.
Sep 09 17:59:50 cherry shairport-sync[1420]: output_rate is 44100.
Sep 09 17:59:50 cherry shairport-sync[1420]: output_format is 3 (0-unknown, 1-S8, 2-U8, 3-S16, 4-S24, 5-S24_3LE, 6-S24_3BE, 7-S32).
Sep 09 17:59:50 cherry shairport-sync[1420]: audio backend desired buffer length is 0.150000 seconds.
Sep 09 17:59:50 cherry shairport-sync[1420]: audio backend latency offset is 0.000000 seconds.
Sep 09 17:59:50 cherry shairport-sync[1420]: audio backend silence lead-in time is -1.000000 seconds. A value -1.0 means use the default.
Sep 09 17:59:50 cherry shairport-sync[1420]: volume range in dB (zero means use the range specified by the mixer): 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: zeroconf regtype is "_raop._tcp".
Sep 09 17:59:50 cherry shairport-sync[1420]: decoders_supported field is 1.
Sep 09 17:59:50 cherry shairport-sync[1420]: use_apple_decoder is 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: alsa_use_hardware_mute is 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: no special mdns service interface was requested.
Sep 09 17:59:50 cherry shairport-sync[1420]: configuration file name "/etc/shairport-sync.conf" resolves to "/etc/shairport-sync.conf".
Sep 09 17:59:50 cherry shairport-sync[1420]: metadata enabled is 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: metadata pipename is "(null)".
Sep 09 17:59:50 cherry shairport-sync[1420]: metadata socket address is "(null)" port 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: metadata socket packet size is "500".
Sep 09 17:59:50 cherry shairport-sync[1420]: get-coverart is 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: convolution is 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: convolution IR file is "(null)"
Sep 09 17:59:50 cherry shairport-sync[1420]: convolution max length 8192
Sep 09 17:59:50 cherry shairport-sync[1420]: convolution gain is 0.000000
Sep 09 17:59:50 cherry shairport-sync[1420]: loudness is 0.
Sep 09 17:59:50 cherry shairport-sync[1420]: loudness reference level is -20.000000
Sep 09 17:59:50 cherry shairport-sync[1420]: disable resend requests is off.
Sep 09 17:59:50 cherry shairport-sync[1420]: diagnostic_drop_packet_fraction is 0.000000. A value of 0.0 means no packets will be dropped deliberately.
Sep 09 17:59:50 cherry shairport-sync[1420]: shutdown requested...
Sep 09 17:59:50 cherry shairport-sync[1420]: Request to shut down all rtsp conversation threads
Sep 09 17:59:50 cherry shairport-sync[1420]: asking playing threads to stop
Sep 09 17:59:50 cherry sudo[1410]: pam_unix(sudo:session): session closed for user root
Sep 09 17:59:50 cherry systemd[1]: shairport-sync.service: Service RestartSec=100ms expired, scheduling restart.
Sep 09 17:59:50 cherry systemd[1]: shairport-sync.service: Scheduled restart job, restart counter is at 1.
Sep 09 17:59:50 cherry systemd[1]: Stopped ShairportSync AirTunes receiver.
```
This will repeat several times.

When it does start successfully, the "free(): double free detected in tcache 2" line doesn't show up:
```
Sep 09 18:06:42 cherry systemd[1]: Starting ShairportSync AirTunes receiver...
Sep 09 18:06:42 cherry shairport-sync[1515]: alsa output device name is "default".
Sep 09 18:06:42 cherry shairport-sync[1515]: Version: "3.2.2-OpenSSL-Avahi-ALSA-pa-dummy-stdout-pipe-soxr-convolution-metadata-sysconfdir:/etc"
Sep 09 18:06:42 cherry shairport-sync[1515]: statistics_requester status is 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: daemon status is 1.
Sep 09 18:06:42 cherry shairport-sync[1515]: daemon pid file path is "/run/shairport-sync/shairport-sync.pid".
Sep 09 18:06:42 cherry shairport-sync[1515]: rtsp listening port is 5000.
Sep 09 18:06:42 cherry systemd[1]: Started ShairportSync AirTunes receiver.
Sep 09 18:06:42 cherry shairport-sync[1515]: udp base port is 6001.
Sep 09 18:06:42 cherry shairport-sync[1515]: udp port range is 100.
Sep 09 18:06:42 cherry shairport-sync[1515]: player name is "Cherry".
Sep 09 18:06:42 cherry shairport-sync[1515]: backend is "(null)".
Sep 09 18:06:42 cherry shairport-sync[1515]: on-start action is "(null)".
Sep 09 18:06:42 cherry shairport-sync[1515]: on-stop action is "(null)".
Sep 09 18:06:42 cherry shairport-sync[1515]: wait-cmd status is 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: on-start returns output is 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: mdns backend "(null)".
Sep 09 18:06:42 cherry shairport-sync[1515]: stuffing option is "0" (0-basic, 1-soxr).
Sep 09 18:06:42 cherry shairport-sync[1515]: resync time is 0.050000 seconds.
Sep 09 18:06:42 cherry shairport-sync[1515]: allow a session to be interrupted: 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: busy timeout time is 120.
Sep 09 18:06:42 cherry shairport-sync[1515]: drift tolerance is 0.001995 seconds.
Sep 09 18:06:42 cherry shairport-sync[1515]: password is "(null)".
Sep 09 18:06:42 cherry shairport-sync[1515]: ignore_volume_control is 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: volume_max_db is not set
Sep 09 18:06:42 cherry shairport-sync[1515]: playback_mode is 0 (0-stereo, 1-mono, 1-reverse_stereo, 2-both_left, 3-both_right).
Sep 09 18:06:42 cherry shairport-sync[1515]: disable_synchronization is 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: use_mmap_if_available is 1.
Sep 09 18:06:42 cherry shairport-sync[1515]: output_rate is 44100.
Sep 09 18:06:42 cherry shairport-sync[1515]: output_format is 3 (0-unknown, 1-S8, 2-U8, 3-S16, 4-S24, 5-S24_3LE, 6-S24_3BE, 7-S32).
Sep 09 18:06:42 cherry shairport-sync[1515]: audio backend desired buffer length is 0.150000 seconds.
Sep 09 18:06:42 cherry shairport-sync[1515]: audio backend latency offset is 0.000000 seconds.
Sep 09 18:06:42 cherry shairport-sync[1515]: audio backend silence lead-in time is -1.000000 seconds. A value -1.0 means use the default.
Sep 09 18:06:42 cherry shairport-sync[1515]: volume range in dB (zero means use the range specified by the mixer): 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: zeroconf regtype is "_raop._tcp".
Sep 09 18:06:42 cherry shairport-sync[1515]: decoders_supported field is 1.
Sep 09 18:06:42 cherry shairport-sync[1515]: use_apple_decoder is 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: alsa_use_hardware_mute is 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: no special mdns service interface was requested.
Sep 09 18:06:42 cherry shairport-sync[1515]: configuration file name "/etc/shairport-sync.conf" resolves to "/etc/shairport-sync.conf".
Sep 09 18:06:42 cherry shairport-sync[1515]: metadata enabled is 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: metadata pipename is "(null)".
Sep 09 18:06:42 cherry shairport-sync[1515]: metadata socket address is "(null)" port 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: metadata socket packet size is "500".
Sep 09 18:06:42 cherry shairport-sync[1515]: get-coverart is 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: convolution is 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: convolution IR file is "(null)"
Sep 09 18:06:42 cherry shairport-sync[1515]: convolution max length 8192
Sep 09 18:06:42 cherry shairport-sync[1515]: convolution gain is 0.000000
Sep 09 18:06:42 cherry shairport-sync[1515]: loudness is 0.
Sep 09 18:06:42 cherry shairport-sync[1515]: loudness reference level is -20.000000
Sep 09 18:06:42 cherry shairport-sync[1515]: disable resend requests is off.
Sep 09 18:06:42 cherry shairport-sync[1515]: diagnostic_drop_packet_fraction is 0.000000. A value of 0.0 means no packets will be dropped deliberately.
Sep 09 18:06:42 cherry sudo[1506]: pam_unix(sudo:session): session closed for user root
Sep 09 18:06:43 cherry shairport-sync[1515]: avahi: service '7B75D91D2856@Cherry' successfully added.
```

```
Raspberry Pi 4 Model B Rev 1.1n

PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
```

**Comment by jordanryanmoore** [comment]
I’ve got the same issue when trying to override the `general.name` option — also running the latest from Raspian.

**Comment by mikebrady** [comment]
Thanks for the post. That is a bug alright, and it has been fixed in recent versions of Shairport Sync. The version in Raspbian, unfortunately, still contains it. 

The updated version is now in Debian backports, so it'll be a matter of time before it becomes available in the Raspbian package system. Alternatively, you could build the latest version from here. Instructions are at [INSTALL.md](https://github.com/mikebrady/shairport-sync/blob/master/INSTALL.md). If you do so, be sure to pay close attention to cleaning up before building and installing, as detailed on the page.

**Comment by jordanryanmoore** [comment]
Manually installing the latest version from master solved it. Thanks. :)

**Comment by BrainStone** [comment]
Sadly still not updated. Is there a way to reach out to the repo maintainers and urge them to update due to it having a breaking bug?

**Comment by el1-x** [comment]
I encountered an issue similar to this and #1052 where Shairport-sync would not start if I manually edited the name="" in the /etc/shairport-sync.conf file.

I could however, start Shairport-sync manually with the --name= option and it was fine. 

My issue at least, was resolved by removing v3.2 found in the raspiOS repo and installing manually.
